dist/htmlminifier.js

```diff
@@ -848,7 +857,9 @@
       return attrValue;
     }
     else if (isMetaViewport(tag, attrs) && attrName === 'content') {
-      attrValue = attrValue.replace(/1\.0/g, '1').replace(/\s+/g, '');
+      attrValue = attrValue.replace(/\s+/g, '').replace(/[0-9]+\.[0-9]+/g, function(num) {
+        return (+num).toString();
```

Comment:
Shouldn't `num` already be a string? This is redundant double-conversion, it seems.